packages/toolchain/elemental-cli/collection.yaml

```diff
@@ -3,15 +3,15 @@ packages:
     name: "elemental-cli"
     bin_name: "elemental"
     category: "toolchain"
-    version: 0.20220415-1
+    version: 0.20220419-1
     fips: false
     labels:
       github.repo: "elemental"
       github.owner: "rancher-sandbox"
       autobump.revdeps: "true"
       autobump.strategy: "git_hash"
       autobump.git.branch: "main"
-      git.hash: "3cf16176734fcb2b02a0f040c929377fe1c7c11e"
+      git.hash: "50e19a94cba7245dae92e3924c05944cc7c4a049"
```

Comment:
```suggestion
      git.hash: "edc412b0dd1c219a07e023d16cafb6251f7280cd"
```

Reply:
The CI failed.. before retriggering, maybe we can include the latest fixes to avoid doubling CI ?

Reply:
Fine with me :+1: